articles/site-recovery/avs-tutorial-replication.md

```diff
@@ -81,15 +81,15 @@ All of these components are installed together on a single Azure VMware Solution
 ## Import the template in VMware
 
 
-1. Sign in to the VMware vCenter server or vSphere ESXi host with the VMware vSphere Client.
+1. Sign in to the VMware vCenter Server with the VMware vSphere Client.
 2. On the **File** menu, select **Deploy OVF Template** to start the **Deploy OVF Template Wizard**.
 
-     ![Screenshot of the Deploy OVF template command in the VMWare vSphere Client.](./media/vmware-azure-tutorial/vcenter-wizard.png)
+     ![Screenshot of the Deploy OVF template command in the VMware vSphere Client.](./media/vmware-azure-tutorial/vcenter-wizard.png)
 
 3. On **Select source**, enter the location of the downloaded OVF.
 4. On **Review details**, select **Next**.
 5. On **Select name and folder** and **Select configuration**, accept the default settings.
-6. On **Select storage**, for best performance select **Thick Provision Eager Zeroed** in **Select virtual disk format**.
```

Comment:
"Thick provision" is recommended to the customers for fixed allocated space. So, we should use the same storage here.